spec/index.bs

```diff
@@ -10359,6 +10388,36 @@ dictionary GPURenderPassColorAttachment {
     where |descriptor| is |view|.{{GPUTextureView/[[descriptor]]}}.
 </div>
 
+<div algorithm>
+    <dfn abstract-op>Calculating total color attachment depth</dfn>(|formats|)
+
+    **Arguments:**
+
+    - sequence<{{GPUTextureFormat}}> |formats|
+
+    **Returns:** {{GPUSize32}}
+
+    1. Let |total| be 0.
+    1. For each |format| in |formats|
```

Comment:
nit: If this algorithm handled nulls, e.g. something like `For each non-null |format| in |formats|`, it would make the callers simpler because 2 of 3 of them wouldn't have to filter out nulls and could just pass the input directly through.

Reply:
it would be 1 of the 3 of them, but you're right.